.github/workflows/reusable-operator-hub-release.yaml

```diff
@@ -0,0 +1,91 @@
+name: Reusable - Create operator hub pull request
+
+on:
+  workflow_call:
+    inputs:
+      org:
+        type: string
+        required: true
+      repo:
+        type: string
+        required: true
+
+jobs:
+  create-operator-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
```

Comment:
we should as well set  installation_id: PUBLISH_APP_INSTALLATION_ID

Reply:
Where did you get `PUBLISH_APP_INSTALLATION_ID`, I don't see where is mentioned on the documentation but I maybe missing something.

Reply:
ping?

Reply:
Correct it does not seem to be mentioned in https://github.com/tibdex/github-app-token, maybe perhaps as `installation_retrieval_mode`, but probably not. 

The PUBLISH_APP_INSTALLATION_ID is defined as a repository secret, but perhaps we don't have to use it.